docs/source/tracking.rst

```diff
@@ -429,6 +429,41 @@ Autologging captures the following information:
 |           |                        | `OneCycleScheduler`_ callbacks                           |               |                                                                                                                                                                       |
 +-----------+------------------------+----------------------------------------------------------+---------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Pytorch (experimental)
+--------------------------
+
+Call :py:func:`mlflow.pytorch.autolog` before your training code to enable automatic logging of metrics and parameters. See example usages with `Pytorch <https://github.com/chauhang/mlflow/tree/master/examples/pytorch/MNIST>`_.
+
+In the current implementation, PyTorch autolog works with the `Lightning training loop <https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/trainer/training_loop.py>`_. The respective metrics associated with ``EarlyStopping Callabacks`` and ``pytorch_lightning.trainer`` are automatically logged. It must be noted that in case of a multi-optimizer scenario (such as usage of autoencoder) by default only the parameters for the first optimizer would be logged.
```

Comment:
@karthik-77 Please move the Optimizer note to the "Note" callout at the end (after the table)

Reply:
Yes i have moved the Optimizer  note to the "Note" Callout at the end